src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -218,6 +221,31 @@ public void setAssistantMode(boolean assistantMode) {
             String.valueOf(assistantMode));
   }
 
+  /**
+   * Sets a collapsible panel collapsed.
+   */
+  public void setCollapsed(@NotNull String title) {
+    applicationPropertiesManager.setValue(A_PLUS_COLLAPSED_PANELS.getName(),
+        getCollapsed() != null ? getCollapsed() + ";" + title : title);
```

Comment:
Remark: this kind of logic could be handled in a separate (domain oblivious) class (that ideally would implement `Set<String>` interface). That could then be easily unit-tested. Just consider! :)